src/repositories/releases/release.module.ts

```diff
@@ -8,10 +8,11 @@ import { ReleaseService } from './release.service';
 @Module({
   imports: [
     MongooseModule.forFeature([
-      { name: 'Releases', schema: ReleaseSchema },
+      { name: 'Release', schema: ReleaseSchema },
       { name: 'Repository', schema: RepositorySchema },
```

Comment:
we should update the names here too to use the Release.name pattern

Reply:
Done for both